sunpy/net/helioviewer.py

```diff
@@ -88,12 +132,13 @@ def get_closest_image(self, date, **kwargs):
 
         return response
 
-    def download_jp2(self, date, directory=None, overwrite=False, **kwargs):
+    def download_jp2(self, date, observatory=None, instrument=None, detector=None,
+                            measurement=None, directory=None, overwrite=False, **kwargs):
```

Comment:
I think the indentation here for `measurement` needs to be moved back to line up with `self`.